sql/moz-fx-data-shared-prod/search_terms_derived/sanitation_job_metadata/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+---
+friendly_name: Sanitized Firefox Search Terms
+owners:
+  - ctroy@mozilla.com
+labels:
+  incremental: true
+description: |-
+  A sanitized variant of search terms submitted to Firefox.
+
+  This table allows removal of PII and slightly longer retention than
+  the underlying ping table.
+scheduling:
```

Comment:
The scheduling block should be removed, since it's related to airflow dags (that is, unless there is some metadata we could put here to indicate it's being updated by a separate mechanism, @jklukas might know).